hiring-developers-with-readthedocs.rst

```diff
@@ -78,6 +78,9 @@ Focusing on your best attributes shows you care about your developers
 and helps you appeal to people who care about your products and mission.
 
 
-**If your company is recruiting** `get in touch`_ **and let us help.**
+.. admonition:: Is your company hiring?
 
-.. _get in touch: mailto:ads@readthedocs.org
+    `Learn more`_ about how Read the Docs can promote your job openings to qualified candidates
```

Comment:
Should link this with a bette title. Like "Hire developers with Read the Docs"

Reply:
Hiring is in the heading immediately above it so I'm not sure about that one specifically. However, the copy is definitely a little "we-focused". Perhaps something like:

```
Are you hiring?

Find qualified candidates already using your tech by promoting your job openings with Read the Docs.
```

I should read [this post](https://stripe.com/atlas/guides/landing-page-copy) every single time I'm working on landing pages and CTAs.

Reply:
I'm mostly thinking about it from SEO perspective. "Learn more" is not a great title since it won't let google know our page is out hiring developers.